src/hooks/nostr/useMessageModeration.js

```
@@ -0,0 +1,216 @@
+      };
+
+      const subscription = ndk.subscribe(filter, {
+        closeOnEose: false,
+        cacheUsage: NDKSubscriptionCacheUsage.CACHE_FIRST,
+      });
+
+      subscription.on('event', (event) => {
+        try {
+          // Extract the message ID being hidden from the 'e' tag
+          const messageIdTag = event.tags.find(tag => tag[0] === 'e');
+          if (messageIdTag && messageIdTag[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)